addOns/oast/src/main/resources/org/zaproxy/addon/oast/resources/Messages.properties

```diff
@@ -60,6 +60,9 @@ oast.panel.table.column.referer=Referer
 oast.panel.currentState.tooltip.lastPolled = {0}: Last Polled Time
 oast.panel.currentState.lastPoll= {0}: {1}
 
+oast.popup.menu.insertPayload=Insert OAST Payload
+oast.popup.menu.new=New
```

Comment:
This can be removed, no longer in use.

Reply:
`New` is not in use I removed that. `insertPayload` is still begin used

Reply:
Yeah, I was referring only to the latter, unfortunately the UI does not make that obvious.